docs/reference-docs/rug/rug-conventions.md

```diff
@@ -28,22 +26,23 @@ concrete examples.
 
 [travis-editors]: https://github.com/atomist-rugs/travis-editors
 
-### Rug Archive
+### Rug Projects
```

Comment:
If we are changing Rug archive to Rug project, it will have to be changed throughout the docs.